src/badges/src/Badge.js

```diff
@@ -36,20 +46,31 @@ class Badge extends PureComponent {
   }
 
   render() {
-    const { theme, color: propsColor, isSolid, ...props } = this.props
+    const {
+      theme,
+      className,
+      color: propsColor,
+      isInteractive,
+      isSolid,
+      ...props
+    } = this.props
 
     const { color, backgroundColor } = theme.getBadgeProps({
-      isSolid,
-      color: propsColor
+      color: propsColor,
+      isSolid
     })
 
+    const appearance = isInteractive ? 'interactive' : 'default'
```

Comment:
Instead of having a different appearance. What about using a `data-interactive` tag so we can use a selector instead?

Reply:
We could do that too. I think this is a little more straightforward in terms of "are interactive styles applied or not"